src/Sylius/Bundle/CoreBundle/Installer/Setup/LocaleSetup.php

```diff
@@ -100,4 +111,16 @@ private function getLanguageName(string $code): ?string
             return null;
         }
     }
+
+    private function updateLocaleParameter(string $code): void
+    {
+        if ($this->filesystem->exists($this->localeParameterFilePath) === false) {
+            throw new \RuntimeException(sprintf('File %s does not exist', $this->localeParameterFilePath));
```

Comment:
It means that not passing the `localeParameterFilePath` (which is deprecated, but still possible) and not having `config/parameters.yaml` file would result in an exception. It could probably happen for a lot of developers. How about just skipping this parameter set if the file is not there? We could write here an information we had before (`'<info>You may also need to add this locale into config/services.yaml configuration.</info>'`)